energy_communities_cooperator/config.py

```diff
@@ -1,4 +1,193 @@
+from odoo.tools.translate import _
+
 COOP_SHARE_PRODUCT_CATEG_REF = "cooperator.product_category_company_share"
 COOP_VOLUNTARY_SHARE_PRODUCT_CATEG_REF = (
     "energy_communities_cooperator.product_category_company_voluntary_share"
 )
+MAPPING__SUBSCRIPTION_MODE__PRODUCT_CATEG_REF = {
+    "member": COOP_SHARE_PRODUCT_CATEG_REF,
+    "company_member": COOP_SHARE_PRODUCT_CATEG_REF,
+    "invited": COOP_SHARE_PRODUCT_CATEG_REF,
+    "company_invited": COOP_SHARE_PRODUCT_CATEG_REF,
+    "voluntary": COOP_VOLUNTARY_SHARE_PRODUCT_CATEG_REF,
+    "company_voluntary": COOP_VOLUNTARY_SHARE_PRODUCT_CATEG_REF,
+}
+MAPPING__SUBSCRIPTION_MODE__DEFAULT_PAGE_TITLE = {
```

Comment:
I don't understand why should this code be in two strings instead of one.
`_("Become Cooperator of {company_name} ") + _("member_eu_ES")`